examples/ensemble/plot_feature_transformation.py

```diff
@@ -112,3 +103,18 @@ def gradient_apply(clf, X):
 plt.title('ROC curve')
 plt.legend(loc='best')
 plt.show()
+
+plt.figure(2)
+plt.xlim(0, 0.2)
+plt.ylim(0.8, 1)
+plt.plot([0, 1], [0, 1], 'k--')
+plt.plot(fpr_rt_lm, tpr_rt_lm, label='RT + LR')
+plt.plot(fpr_rf, tpr_rf, label='RF')
+plt.plot(fpr_rf_lm, tpr_rf_lm, label='RF + LR')
+plt.plot(fpr_grd, tpr_grd, label='GBT')
+plt.plot(fpr_grd_lm, tpr_grd_lm, label='GBT + LR')
+plt.xlabel('False positive rate')
+plt.ylabel('True positive rate')
+plt.title('ROC curve')
```

Comment:
Maybe change the title to make it explicit that this is a zoom on on the top left corner of the ROC curves.